crates/primitives/Cargo.toml

```diff
@@ -32,6 +32,10 @@ std = [
     "scale-info/std",
 ]
 
+# The feature is not yet utilized in this crate, but we need to add it
+# here already anyway, since the CI tries to execute fuzzing on each crate.
+ink-fuzz-tests = ["std"]
+
```

Comment:
Maybe we should adjust the `.gitlab-ci.yml` file to the system instead of the system under test to our `.gitlab-ci.yml` file. :upside_down_face: